### ISSUE TITLE
2. Create ability for qa to talk to Lark

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#197]
# Summary

blocked by #196 

# Acceptance Criteria

- [ ]

# Screenshots or Video

# Testing Instructions

# Notes

Controlled values meeting:
https://docs.google.com/document/d/1rLDHufesIWh71unkeAeucVpPy5syFaeGWQXr3KucIfQ/edit

**Comment by ShanaLMoore** [comment]
Lark's purpose is to be intermediary between the surfliner project and an authority (like library of congress). Populate library of congress into lark w adjustments for better terminology and so that it can get cached until a necessary update someday.

Lark may need to be its own application, with it's own docker setup that UTK will communicate with. Lark has its own database that will need to get populated. 

[lark](https://gitlab.com/surfliner/surfliner/blob/trunk/lark/README.md) 
[lark client](https://gitlab.com/surfliner/surfliner/-/tree/trunk/lark-client)
 
Questioning authority is a query service. Provides an interface. ie: Search the widgets authority for the term "sprocket". QA will find the widget authority (it may have a yml file that it returns, or it'll return data some other way). Creates a consistent way for hyrax application to ask a proxy for an authority for information. 

Lark is a read write application. Will get populated by authorities that QA will return.


## TODO: 
Get QA configured to work with a remote authority. This will prove that QA is configured correctly and able to make remote requests. 

make singular authority proxy for each one. then create a naive aggregate that'll query each of the listed properties. 
ie: LCSH and Wikidata

Lark will be a caching layer between our application and the authority. 

Goal:

**Comment by ShanaLMoore** [comment]
What remote authorities does QA already know how to talk to?

**Comment by jillpe** [comment]
no longer relevant
